services/gateway/apis/http-version3/swagger/parameters/blockchainApps.json

```diff
@@ -14,7 +14,25 @@
         "description": "Chain ID to query (can be expressed as an interval from:to)",
         "type": "string",
         "format": "id",
-        "minLength": 1
+        "minLength": 8,
+        "maxLength": 8
+    },
+    "chainIDWithoutInterval": {
+        "name": "chainID",
+        "in": "query",
+        "description": "Chain ID to query",
+        "type": "string",
+        "format": "id",
+        "minLength": 8,
+        "maxLength": 8
+    },
+    "chainIDCSV": {
+        "name": "chainID",
+        "in": "query",
+        "description": "Chain ID to query (can be expressed as CSV)",
+        "type": "string",
+        "format": "id",
+        "minLength": 8
     },
```

Comment:
Do we need duplicate entries here? Can we use only one of them and name it `chainID`?

Reply:
there are three versions of the description:

1. Chain ID to query (can be expressed as an interval from:to)
2. Chain ID to query (can be expressed as CSV)
3. Chain ID to query

Reply:
Please remove the entry for `Chain ID to query (can be expressed as an interval from:to)` since this is invalid now. And, kindly ensure to add `.` at the end of the descriptions